engines/wintermute/base/base_persistence_manager.cpp

```diff
@@ -821,6 +821,64 @@ bool BasePersistenceManager::transferVector2(const char *name, Vector2 *val) {
 	}
 }
 
+#ifdef ENABLE_WME3D
+//////////////////////////////////////////////////////////////////////////
+// Vector3
+bool BasePersistenceManager::transferVector3d(const char* name, Math::Vector3d* val) {
```

Comment:
These could probably be implemented in terms of calls to transferFloat(), which may or may not be cleaner.

Reply:
I'll leave it as is since it is consistent with transferVector2 this way. Fixed the pointer formatting, though.